src/documentation/basic-opts.page

```diff
@@ -12,17 +12,17 @@ This document covers the basic operations provided by Uniqush. It won't cover th
+Uniqush runs as a stand alone service, which means you don't need to write any code to link against uniqush. Uniqush listens on a TCP port when it is running. You can change the port in its configuration file. To send request to uniqush, you simply need to connect to that port, send an HTTP POST request with the parameters inside the body.
 
 ## Adding Push Service Providers to a Service
 
-Without telling uniqush infomation about push service provider, a server can never send data through uniqush. Usually, to use a push service provider, uniqush need to know the type of that push service provider (e.g. [GCM], [APNS], Black Berry Push Service or Microsoft Push Notification Service for Windows Phone) and authentication information -- for example, authentication token for [GCM]; private/public keys for [APNS], etc.
+Without telling uniqush infomation about a push service provider, a server can never send data through uniqush. Usually, to use a push service provider, uniqush need to know the type of that push service provider (e.g. [GCM], [APNS], Black Berry Push Service or Microsoft Push Notification Service for Windows Phone) and authentication information -- for example, authentication token for [GCM]; private/public keys for [APNS], etc.
 
 You can add more than one push service provider information to a service, so that you can support multiple platforms.
 
-## Subscribe a Service
+## Subscribing to a Service
 
-When a subscriber subscribes a service, uniqush need to know at least three things: the subscriber's name; the service name; and the delivery point information. The delivery point information including the identity of the device, its OS type and other push service provider specific information. For example, for android delivery points, they need to provide a registration id to uniqush, which will be used to send data through [GCM] later.
+When a subscriber subscribes to a service, uniqush need to know at least three things: the subscriber's id; the service name; and the delivery point information. The delivery point information including the identity of the device, its OS type and other push service provider specific information. For example, for android delivery points, they need to provide a registration id to uniqush, which will be used to send data through [GCM] later.
 
 Sending such information to uniqush could be done by a mobile app, which connect to uniqush server through HTTP and provide the information. NOTE: you can always use a web-server in front of uniqush as a proxy. So that the app can first connect to your web-server and the information will be forwarded to uniqush through HTTP. This is a recommended way, because for security consideration, you may want to set an HTTPS connection instead of HTTP connection between the web-server and the app.
```

Comment:
"web-server" -> "web server"

"So that" - awkward phrasing.

This whole paragraph should probably be rewritten to be clearer. Mention considerations such as "the web server may also ensure that the subscriber id corresponds to the logged in user", "The uniqush API allows listing subscribers and sending pushes, so it should not be used directly", etc.

"a recommended" -> "the recommended"
"consideration" -> "considerations"